File: Part4/Rust/Function_UsedEquivalance.rs
use std::io;

fn main()
{
    let mut user_input = String::new();
    
    println!( "Type 1 and press enter." );
    io::stdin().read_line( &mut user_input ).expect( "Failed to read line" );
    
    // Using == (comparison) instead of = (assignment)
    let entered_integer == user_input.trim().parse::<i32>().unwrap();
    
    println!( "The user entered the integer {}", entered_integer );
}

File: Part4/C#/Function_UsedEquivalance.cs
using System;

class Program
{
    static void Main()
    {
        string userInput;

        Console.WriteLine("Type 1 and press enter.");
        userInput = Console.ReadLine();

        // Using == (comparison) instead of = (assignment)
        int enteredInteger == int.Parse(userInput);

        Console.WriteLine("The user entered the integer {0}", enteredInteger);
    }
}
